memory/replay_buffer.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
```

Comment:
What is the difference between `buffer.py` and `replay_buffer.py`.

Reply:
The naming of `buffer.py` is somewhat confusing, since it is a fixed dataset rather than a dynamic buffer. I'll change the name of `buffer.py`.